app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts.order(created_at: :desc)
+    @posts = @user.posts.includes(:comments)
   end
 
```

Comment:
- [ ] `posts#create`, `comments#create`, and `likes#create` actions uses flash messages for a successful and not successful path. Currently, the message is not being rendered in the view. Please fix it.